console-subscriber/src/lib.rs

```diff
@@ -406,6 +406,7 @@ where
     fn on_event(&self, event: &tracing::Event<'_>, ctx: Context<'_, S>) {
         let metadata = event.metadata();
         if self.waker_callsites.contains(event.metadata()) {
+            tracing::debug!(?event, "waker op event");
```

Comment:
Aren't these problematic for triggering an event inside an event handler?

Reply:
whoops, meant to remove these!